src/main/resources/quilt.mod.json

```diff
@@ -114,6 +114,10 @@
 				"id": "quilted_fabric_rendering_v0",
 				"optional": true
 			},
+			{
+				"id": "quilted_fabric_models_v0",
+				"optional": true
+			},
```

Comment:
once again, i wish this was alphabetically sorted; but i can address that post-merge :p